### PR TITLE
GH-5876 Make diff output from gofmt readable when running make check-server-style

### PR DESCRIPTION
**Pull request by bradhowes**
#### Summary
Make diff output from gofmt readable when running `make check-server-style`

#### Ticket Link
#5876

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -198,13 +198,14 @@ check-client-style:
 
 check-server-style: govet
 	@echo Running GOFMT
-	$(eval GOFMT_OUTPUT := $(shell gofmt -d -s api/ model/ store/ utils/ manualtesting/ einterfaces/ cmd/platform/ 2>&1))
-	@echo "$(GOFMT_OUTPUT)"
-	@if [ ! "$(GOFMT_OUTPUT)" ]; then \
-		echo "gofmt success"; \
-	else \
+	@gofmt -d -s api/ model/ store/ utils/ manualtesting/ einterfaces/ cmd/platform/ 2>&1 > gofmt_output.log
```
Comment (crspeller) [inline Makefile]: Can you add gofmt_output.log to the gitignore?

File: api/websocket_test.go [hunk 2]
```diff
@@ -319,6 +319,10 @@ func TestWebsocketOriginSecurity(t *testing.T) {
 
 	url := "ws://localhost" + utils.Cfg.ServiceSettings.ListenAddress
 
+	defer func() {
```
Comment (crspeller) [inline api/websocket_test.go]: This seems unrelated.

**Comment by esethna** [comment]
Thanks @bradhowes, appreciate the pull request!  Just letting you know that there may be a delay in reviewing and merging your PR since we are approaching code complete for v3.8, releasing April 16th. Please see our [release process](https://docs.mattermost.com/process/release-process.html) for more details.

**Comment by jasonblais** [comment]
Thanks for the pull request @bradhowes! Would you be able to review the notes included above?

**Comment by lindy65** [comment]
Hi @bradhowes, we haven't received an update so we'll assume this pull request  is no longer being worked on and will close it due to inactivity.

We will keep your branch open, so you can start where you left off if you’re still interested working on it.

If you have any questions, please don’t hesitate to let us know.
